### ISSUE TITLE
adding unique eventId

### ISSUE THREAD

**Issue by JulianGerman** [builttoroam/device_calendar#414]
How can I add custom eventId from f.e secure storage when i'm adding new Event to Calendar? I need it to add features like deleting event by Id. In class Event there is property eventId, read-only and i can't set value to this property.

**Comment by thomassth** [comment]
I'm unsure about it.

I think our event IDs are set by the system itself, which doesn't seem to accept custom IDs.

https://developer.android.com/guide/topics/providers/calendar-provider

It would be very useful to set custom IDs, but we might need to do an extra layer of conversion, which can be slow and/or troublesome.

**Comment by JulianGerman** [comment]
Yes, i know that your event Id's is a read-only property, but how can we read it? when I want to print it after adding event, the event Id is null so i haven't any option to get this value to delete this event in the future.
